packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -337,6 +431,52 @@ See [ID Token Claims](/docs/reference/api/oidc/#id-token-claims) for a list of I
 }
 ```
 
+You can also use the `replace` operation to modify members within JSON objects and elements within arrays. For example, you have a JSON object in a claim called `employee_profile`, and you want to update the email address of the employee.
+
+The existing target JSON object:
+```json
+{
+"employee_profile": {
```

Comment:
nit: should this line have 4 spaces at the beginning or website engine will format it well?

Reply:
Web site engine formats it

Reply:
the website engine formats